modules/virt-enabling-virt-repos.adoc

```diff
@@ -20,12 +20,12 @@ Enable the {VirtProductName} repository for your version of {op-system-base-full
 +
 [source,terminal]
 ----
-# subscription-manager repos --enable cnv-4.12-for-rhel-8-x86_64-rpms
+# subscription-manager repos --enable cnv-4.13-for-rhel-8-x86_64-rpms
```

Comment:
@sheriff-rh This is virt content. The repos for virtctl installation need to be verified, so we have our own task (CNV-18933) to cover it. Thanks.